spec.bs

```diff
@@ -296,21 +296,46 @@ To <dfn>look up the key commitments</dfn> for a given [=/origin=] |issuer|, run
 
 Note: |pstVersion| is a string identifier representing different cryptographic versions of tokens that can be used with this API. User agents should only select keys for versions they support, ordered by which versions they prefer based on performance and any user defined preferences.
 
+To <dfn>look up latest keys</dfn> for a given |commitment|, run the following steps:
```

Comment:
|commitment| needs a type here and it looks like you don't have one for that yet. Two options:

1. Modify this algorithm to take an `[=/origin=]` and look up the key commitments for that origin.
2. Modify https://wicg.github.io/trust-token-api/#issuer-public-keys to say something like

```
A <dfn>key commitment</dfn> is a [=map=] representing a collection of cryptographic keys...
```

then you can pass `[=key commitment=]` around.

Thinking about it, doing both actually seems optimal to me?

Reply:
You can then also say `a map of [=key commitments=]` in https://wicg.github.io/trust-token-api/#pstkeycommitments

Reply:
Defined it and linked to it, not sure if its the right format.